src/App.tsx

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+type Props = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+};
+
+function filterMovies(query: string, films: Props[]) {
+  const lowCaseQuery = query.toLowerCase();
+  const visibleMovies
+    = films.filter((movie) => movie.title.toLowerCase().includes(lowCaseQuery)
+    || movie.description.toLowerCase().includes(lowCaseQuery));
+
+  return visibleMovies;
+}
```

Comment:
make it looks more clearly
```suggestion
function filterMovies(query: string, films: Props[]) {
  const includesQuery = (string) => string.toLowerCase().includes(query.toLowerCase())
  
  const visibleMovies
    = films.filter(({ title, description }) => includesQuery (title) || includesQuery(description));

  return visibleMovies;
}
```